Formula/arturo.rb

```diff
@@ -0,0 +1,19 @@
+class Arturo < Formula
+  desc "Simple, modern and portable programming language for efficient scripting"
+  homepage "http://arturo-lang.io"
+  url "https://github.com/arturo-lang/arturo/archive/v0.9.4.6.tar.gz"
+  sha256 "90dfd8870ab0bfadd14cd611a33353a1d4b66b25e0c56d75cab895989be25c98"
+  license "MIT"
+
+  depends_on "gmp" => :build
+  depends_on "nim" => :build
+
+  def install
+    system "/bin/sh", "build.sh"
+    bin.install "bin/arturo"
+  end
+
+  test do
+    system "#{bin}/arturo", "-v"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks for the clarification. I have already fixed that. :)